Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -731,35 +793,37 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::Load(const u32 stage)
     InvalidateTexture(oldest_entry);
   }
 
+  // Loaded texture data (which may not match requested texture data)
+  u32 width = request.Width();
+  u32 height = request.Height();
+  u32 stride = request.m_levels[0].padded_width;
+
   std::shared_ptr<HiresTexture> hires_tex;
   if (g_ActiveConfig.bHiresTextures)
```

Comment:
One thing about this is I really would like to think we could improve the clarity here...there's so much hopping around based on if we're dealing with highres or "lowres" textures.  Could we instead break it down to something like this:

```
if (g_ActiveConfig.bHiresTextures)
{
  CalculateHiResTextures(entry);
}
else
{
  CalculateLowResTextures(entry);
}

return ReturnEntry(stage, entry);
```

Not sure what all parameters we would need to make this work.